docs/guides/self-signed-ca-certs.md

```diff
@@ -0,0 +1,60 @@
+---  
+title: Fetch Packages from Registry with Self-Signed CA Certificate  
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Overview of Crossplane for Registry with Self-Signed CA Certificate  
+
+> ! Using self-signed certificates in production requires proper certificate  
+infrastructure and process. Without deep knowledge of this topic, it is 
+recommended to only use self-signed certificates for testing.
+
+Crossplane loads the default system certificates. This precludes fetching 
+packages from private registries that may use certs from a self-signed CA.
+
+The default transport can be overridden with one that passes additional certs
+to the TLSConfig. This allows for self-signed certificate to be
+trusted.
+
+Crossplane needs to be installed via the Helm chart with the 
+`registryCaBundleConfig.name` and `registryCaBundleConfig.key` parameters 
+defined. See [Install Crossplane](https://crossplane.io/docs/v1.6/reference/install.html#uninstalling-the-chart).
```

Comment:
I think this missed getting updated

Reply:
sorry, looking through the past comments, not finding it. what did we want here?

Reply:
Ah my apologies :/ Looks like my comment didn't get posted. Could you change this to reference style (we have standardized on this across Crossplane docs), and also make it a relative link so that it points to the same version of the documentation that the user is viewing this guide with?

Reply:
doh, I think I got it. On side note, would be great if `latest` worked in doc links so we don't have to keep updating these things when CP revs. Or maybe just an automation that replaces a tag with latest docs.